site-content/source/modules/ROOT/pages/community.adoc

```diff
@@ -45,70 +45,94 @@ Read the https://www.apache.org/foundation/policies/conduct[Apache Code of Condu
 == Discussions
 If you are a Cassandra user or want to learn more, we invite you to join these channels created specifically for end user questions and discussions.
```

Comment:
"If you are a Cassandra user or if you want to learn more"

Reply:
Updated 👍